assembl/static2/js/app/graphql/ThematicsDataQuery.graphql

```diff
@@ -0,0 +1,24 @@
+query ThematicsDataQuery($identifier: String!, $lang: String!) {
+  thematicsData: ideas(identifier: $identifier) {
+    ... on IdeaInterface {
+      order
+    }
+    ... on Thematic {
+      id
+      title(lang: $lang)
+    }
+    ... on Idea {
+      id
+      parentId
+      title(lang: $lang)
+    }
+  }
+  rootIdea {
```

Comment:
very important, `rootIdea((identifier: $identifier)` (discussionPhaseId in the future)